pkgs/sdk/server/src/Internal/DataSources/DataSourceOutageTracker.cs

```diff
@@ -96,7 +96,7 @@ private async Task WaitForTimeout(Task outageEnded)
                     return;
                 }
                 var errorsDesc = string.Join(", ", _errorCounts.Select(kv => DescribeErrorCount(kv.Key, kv.Value)));
-                _log.Error("LaunchDarkly data source outage - updates have been unavailable for at least {0} with the following errors: {1}",
+                _log.Error("A streaming connection to LaunchDarkly has not been established within {0} after the connection was interrupted. The following errors were encountered: {1}",
```

Comment:
This probably only makes sense for streaming, but I do want to double check if this is streaming only. Otherwise the message could be interpolated.

Reply:
I thought the same thing, it doesn't seem like it should be streaming specific.  Java currently says "streaming", so I just mimic'd that.  I'll also dig a bit.

Reply:
Removed the data source type from the string.